App.tsx

```diff
@@ -18,38 +18,30 @@ import {
   StatusBar,
 } from 'react-native';
 
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
-
-import {Hello} from './components/Hello';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+import {Button} from './components/Button';
+import {FormTextInput} from './components/FormTextInput';
 
 const App = () => {
-  const usingHermes = typeof HermesInternal === 'object' && HermesInternal !== null;
-
   return (
     <Fragment>
       <StatusBar barStyle="dark-content" />
       <SafeAreaView>
         <ScrollView
           contentInsetAdjustmentBehavior="automatic"
           style={styles.scrollView}>
-          <Header />
-          { usingHermes &&
-            <View style={styles.engine}>
-              <Text style={styles.footer}>Engine: Hermes</Text>
-            </View>
-          }
-          <View style={styles.sectionContainer}>
-            <Text style={styles.sectionTitle}>My Awesome Project</Text>
+          <View style={styles.sectionHeader}>
+            <Text style={styles.sectionTitle}>Bem vindo(a) à Taqtile!</Text>
           </View>
           <View style={styles.body}>
-            <View style={styles.sectionContainer}>
-              <Hello name="World" enthusiasmLevel={1} />
+            <View style={styles.container}>
```

Comment:
Container em react é usado para as camadas que puxam dados
entao a gnt não chama um componente visual de container... geralmente uso "wrapper"

https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0

Reply:
Alterei o alias desse style para "wrapper" ;D